bin/list-nginx-class-ingresses.rb

```diff
@@ -56,6 +56,29 @@ def get_ingresses
   JSON.parse(stdout).fetch("items")
 end
 
+def non_production_tuple(ingress)
+  ingress_name = ingress.dig("metadata", "name")
+  namespace = ingress.dig("metadata", "namespace")
+  if non_production?(namespace)
+    {namespace: namespace, ingress_name: ingress_name}
+  end
+end
+
+def get_namespaces
+  json = `kubectl get namespaces -o json`
+  JSON.parse(json).fetch("items")
+end
+
+def non_production?(namespace)
```

Comment:
There's duplication here from the `production?` method. I'd suggest this:
```
def non_production?(namespace)
  !production?(namespace)
end
```
Having said that, I don't know if this method even needs to exist, since instead of having `non_production?(namespace)` in your code, you could just put `!production?(namespace)`.